actions/upload-to-play-store/action.yml

```diff
@@ -0,0 +1,60 @@
+name: 'Upload to play store'
+description: 'Uploads the supplied aabs to the Google Play Store'
+inputs:
+  aab-paths:
+    description: 'App bundle paths to upload, JSON object'
+    required: true
+  package-name:
+    description: 'App package name'
+    required: true
+  json-key-data:
+    description: 'JSON key data'
+    required: true
+  track:
+    description: 'Google Play track to upload to'
+    required: false
+    default: 'internal'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+
+    - name: Update fastlane and upload to play store
+      run: |
+        export LC_ALL=en_US.utf8
```

Comment:
Should these not be with the other environment variables?
**OPTIONAL**

Reply:
Moved to be with other env variables